ngraph/core/include/ngraph/op/tensor_iterator.hpp

```diff
@@ -30,9 +30,9 @@ namespace ngraph
                 std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
                 /// \return the body of the iteration
-                std::shared_ptr<Function> get_body() const { return m_body; }
+                std::shared_ptr<Function> get_body() const { return m_bodies[0]; }
```

Comment:
As we inherit from MultiSubgraph operation, should we check that several bodies were not created? The same for other operations with 1 body.

Reply:
Good question. I think no, because each  SubGraphOp constructors(for exception SubGraphOp(SubGraphOp&&)) or any derived classes constructors should call MultiSubGraphOp(size_t bodies_index) or MultiSubGraphOp(const OutputVector& args, size_t bodies_index)

Reply:
I think this check might be reasonable in validate_infer_type function of TensorIterator/Loop ops, the size of m_bodies vector should be exactly 1